ExampleMod/Common/Players/ExampleAccessorySlot.cs

```diff
@@ -0,0 +1,35 @@
+﻿using ExampleMod.Content.Items;
+using System.Collections.Generic;
+using Terraria;
+using Terraria.ID;
+using Terraria.ModLoader;
+
+namespace ExampleMod.Common.Players
+{
+	public class ExampleModAccessorySlot1 : ModAccessorySlot
+	{
+		// If the class is empty, everything will default to a basic vanilla slot.
+	}
+
+	public class ExampleModAccessorySlot2 : ModAccessorySlot
+	{
+		// If the class is empty, everything will default to a basic vanilla slot.
+	}
+
```

Comment:
Add CustomDrawAccessorySlot : ModAccessorySlot with overriding XLoc, YLoc and DrawModded